### DISCUSSION TITLE
Workshop exercises comments/questions

### DISCUSSION THREAD

**Discussion by rebecca-vanhoeck** [SAEL-SWFSC/PAM-SI-Mtg-March2024#17] [General]
1. Drafting an introduction and identifying references will be a more time-intensive task than some of the others, should we consider providing some text, a suggested outline, or a list of references ahead of time, so that group can focus more on the github process than the content of the report? 
2. If we are all working on the notes is there a difference between authors and attendees? I think its fine to have both, but am curious about our expectations for these. @shannonrankin

**Comment by shannonrankin** [comment]
Becca-
1. (a) Introduction-- material is there to copy/paste (not asking them to do any writing of their own). I think based on your question I may need to revise the OpenScience detailed explanation of the task so that it is more clear that they will only copy/paste, not write anything themselves? I have instructions to find the documents and what to cut/paste. Please suggest recommendations to make it more clearw. The github list is only basic task. 
1. (b) References-- actually, thank you for bringing this up. I was thinking of asking Sofie to recommend 1 or 2 articles she thinks will most likely be referenced, and then to have them search out those references. Thoughts?

2. Authors vs Attendees-- I think that is a discussion for others. For now, I suggested that only the core group be authors (as part of the exercise), and that everyone (even authors) be added to the Acknowledgements.  Ultimately, this is up to Sofie & the group-- perhaps we can bring it up at the meeting on Wednesday?

**Comment by rebecca-vanhoeck** [comment]
Hi Shannon - Thanks for the quick reply!

1. I was moving too quickly and skimmed right over the instructions to copy the background from the SI proposal. That is perfect, but I will still take a closer review to see if anything could be made more clear.
2. On references - that sounds good to me! I will ask Sofie for recommendations. 
3. That sounds good to me as well about authors vs attendees. We can bring it up this afternoon, but I think that's a good plan.
